lib/error.js

```diff
@@ -75,7 +76,11 @@ AppError.translate = function (response) {
     }
   }
   else if (payload.validation) {
-    error = AppError.invalidRequestParameter(payload.validation)
+    if (payload.message && payload.message.indexOf(messages.any.required) >= 0) {
+      error = AppError.missingRequestParameter(payload.validation.keys)
```

Comment:
According to https://github.com/mozilla/fxa-auth-server/blob/master/docs/api.md#response-format we have not publicly committed to any particular body format for errno 108, so let's change it to talk about `params` plural rather than a single param, and keep the logic you have here.